webidl/publication_link.webidl

```diff
@@ -1,7 +1,7 @@
 dictionary PublicationLink {
-    required DOMString                  url;
-             DOMString                  encodingFormat;
-             DOMString                  name;
-             sequence<DOMString>        rel;
-             sequence<PublicationLink>  children;
+    required DOMString           url;
+             DOMString           encodingFormat;
+             DOMString           name;
```

Comment:
This means that `name` won't be localizable.

Reply:
Changed in https://github.com/w3c/wpub/pull/285/commits/aefe40f27a55ebf37d4272c8d887d32d5bcd2bf3